http/src/test/resources/reference.conf

```diff
@@ -93,6 +93,12 @@ sam {
 
 google {
   leoServiceAccountJsonFile = "fakePath"
+  groups {
+    subEmail = "google@dev.test.firecloud.org"
+    // TODO Change the default values
+    dataprocImageProjectGroupName = "dataproc-image-project-group"
+    dataprocImageProjectGroupEmail = ${google.groups.dataprocImageProjectGroupName}"@test.firecloud.org"
```

Comment:
where does `google.groups.dataprocImageProjectGroupName` come from?

Reply:
I think it's a reference to the previous line in the config